source/features/hide-tips.css

```diff
@@ -1,35 +1,7 @@
-/* Remove the "Styling with Markdown is supported" link when a PR comment is in edit mode */
-a.tabnav-extra[href$='mastering-markdown/'] {
-	display: none !important;
-}
-
-/* Remove the "helpful" banner on the issue tracker listing */
-.repository-content .mb-4.js-notice {
-	display: none !important;
-}
-
-/* Remove the "helpful" popover on the repo label page */
-.labels-list .TutorialPopover {
-	display: none !important;
-}
-
-/* Remove "Continuous integration has not been set up" message on PRs */
-.branch-action-item.marketplace-product-callout {
-	display: none;
-}
-
-/* Remove "pro tip!" box on profile page (appears when name isn't set) */
-.new-user-avatar-cta {
-	display: none !important;
-}
-
 /* Remove random protip at the bottom of some pages */
+.protip, /* Conversations */
+.js-notifications-list-protip, /* "Save" and "Done" notification lists */
 .paginate-container + .text-center:last-child, /* Conversation lists */
-.js-notifications-container .js-navigation-container + .flex-items-center:last-child { /* Notification list */
-	display: none !important;
-}
-
-/* Remove Marketplace marketing box on PRs */
-.js-marketplace-callout-container {
+.js-notifications-container .js-navigation-container + .flex-items-center:last-child { /* Notifications inbox */
```

Comment:
What is this supposed to match? I get no result.

Reply:
![image](https://user-images.githubusercontent.com/46634000/150334526-16c6134c-3259-4428-ad6f-0aff8087b825.png)

(It only appears when you have notification in your Inbox.)

Maybe this would be a better selector though:
```css
.js-notifications-container :is(.octicon-light-bulb, .octicon-light-bulb + div)
```

Reply:
Isn't the DOM like this?

```
.js-notifications-container
	...
		.js-notifications-list
			.Box-body
				.js-navigation-container
		.flex-items-center
```

Reply:
I'm talking about the "Inbox" page